chromium/preloadlist/preloadlist.go

```diff
@@ -99,7 +103,7 @@ func (idx IndexedEntries) Get(domain string) (Entry, HstsPreloadEntryFound) {
 			return entry, AncestorEntryFound
 		}
 	}
-	return Entry{"", "", false}, EntryNotFound
+	return Entry{"", "", false, "custom"}, EntryNotFound
```

Comment:
This function returns an all-zero Entry if nothing is found. The policy should be "" to keep it returning an all-zero Entry.